lib/client.js

```diff
@@ -378,8 +378,15 @@ class AssetComputeClient extends EventEmitter {
         }
         return response;
     }
-}
 
+    async checkEventJournal(journalUrl) {
```

Comment:
"check" is a generic term. What does it "check"?

If it tests whether the journal is ready, we should call it something like

```
isEventJournalReady()
```